install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
+        repo="https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"
```

Comment:
I would declare this at the top of the file and rename it to all caps, to make it clearer for users that they could override it via `export REPO=foobar`

Reply:
I expect there will be other repos for older releases, so I don't think we should declare it at the top. I will however make it overridable.